samples/net/wifi/shell/socs/esp32_procpu.conf

```diff
@@ -1,11 +1,2 @@
-CONFIG_WIFI=y
-
-CONFIG_NETWORKING=y
 CONFIG_NET_L2_ETHERNET=y
-
-CONFIG_NET_IPV6=n
-CONFIG_NET_IPV4=y
-CONFIG_NET_DHCPV4=y
 CONFIG_ESP32_WIFI_STA_AUTO_DHCPV4=y
```

Comment:
@kartben Do you happen to know how do other vendors have its Wi-Fi DHCPv4 IP address automatically? I could get rid of this "CONFIG_ESP32_WIFI_STA_AUTO_DHCPV4=y" in favor of what Zephyr already offers.

Reply:
@jukkar can probably help - I do not know the answer to that question myself

Reply:
It seems that currently the drivers do the DHCP init. Did not investigate very deeply but perhaps we could do this in one place instead of in various drivers.